### PR TITLE
DDF-1224 Updated karaf to version 2.4.3

### PR DESCRIPTION
**Pull request by clockard**
Updated the karaf version to 2.4.3. Updated dependencies as needed. 
@mrmateo @shaundmorris @pklinef please verify these changes. 
@shaundmorris, once approved can you be the committer for this PR?

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf-platform/84)

### CODE REVIEW

File: metrics/platform-metrics-webconsole-plugin/pom.xml [hunk 1]
```diff
@@ -97,6 +97,7 @@
                         <Import-Package>
                             javax.servlet;version="[2.5,3)",
                             javax.servlet.http;version="[2.5,3)",
+                            org.apache.felix.webconsole; version="[3.1.0,4.0.0)",
```
Comment (mrmateo) [inline metrics/platform-metrics-webconsole-plugin/pom.xml]: Did you see an issue with this import when you did the upgrade?

Comment (clockard) [inline metrics/platform-metrics-webconsole-plugin/pom.xml]: Yes. With 2.4.3 the exported version was 3.1.2 but this was looking for the range [3.2.0,4.0.0). Will this be an issue?

Comment (mrmateo) [inline metrics/platform-metrics-webconsole-plugin/pom.xml]: Probably won't be an issue. I was just curious why the change was needed since the export version didn't change between karaf 2.4.1 and 2.4.3.

File: platform-commands/pom.xml [hunk 2]
```diff
@@ -31,7 +31,7 @@
         <dependency>
             <groupId>org.apache.karaf.shell</groupId>
             <artifactId>org.apache.karaf.shell.console</artifactId>
-            <version>2.3.2</version>
```
Comment (shaundmorris) [inline platform-commands/pom.xml]: I like the use of the property as it will help maintain the Karaf version. Leaving older versions can lead to false positives in the OWASP scans as well.

**Comment by clockard** [comment]
@djblue will be the hero for this pr.

**Comment by djblue** [comment]
:+1: I was able to build this PR successfully, and all of the apps started fine on manual inspection.

**Comment by ricklarsen** [comment]
Reviewed 5 of 5 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks broke.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf-platform/84)

**Comment by pklinef** [comment]
:+1:
